h2o-genmodel/src/main/java/hex/genmodel/easy/error/CountingErrorConsumer.java

```diff
@@ -13,13 +13,18 @@
  */
 public class CountingErrorConsumer extends EasyPredictModelWrapper.ErrorConsumer {
 
-  private AtomicLong dataTransformationErrorsCount = new AtomicLong();
+  private Map<String, AtomicLong> dataTransformationErrorsCountPerColumn = new ConcurrentHashMap<>();
   private Map<String, AtomicLong> unknownCategoricalsPerColumn = new ConcurrentHashMap<>();
 
   /**
    * @param model An instance of {@link GenModel}
    */
   public CountingErrorConsumer(GenModel model) {
+
+    for (String column : model.getNames()) {
+      if (!column.equals(model.getResponseName())) dataTransformationErrorsCountPerColumn.put(column, new AtomicLong());
```

Comment:
I would be careful with calling `getResponseName` on unsupervised models. It can return non-sense:

      /** The name of the response column. */
      @Override public String getResponseName() {
        // Note: _responseColumn is not set when deprecated constructor GenModel(String[] names, String[][]     domains) is used
        int r = getResponseIdx();
        return r < _names.length ? _names[r] : _responseColumn;
      }

Reply:
I really want to exclude everything related to response column. So I gave it another try. Plus I realized the offset column should probably not be there too :)